esp/src/src/Session.ts

```diff
@@ -45,7 +45,10 @@ getBuildInfo().then(info => {
     dojoConfig.currencyCode = info["currencyCode"] ?? "";
 });
 
-export const formatCost = d3Format(".2f");
+export function formatCost(value) {
+    const format = d3Format(".2f");
+    return `${format(value)} (${dojoConfig?.currencyCode || "$"})`;
```

Comment:
Evidently ?? doesn't consider empty string something to instead return the right-hand side, so used the || instead

Reply:
Correct...